src/rest-api/endpoints/management-endpoint/search-endpoint.md

```diff
@@ -0,0 +1,240 @@
+`SELECT * FROM Document WHERE dc:title LIKE 'My Title%'`
+
+```curl
+curl -X POST -u Administrator:Administrator \
+http://localhost:8080/nuxeo/api/v1/management/search/reindex?query=SELECT+%2A+FROM+document+WHERE+dc%3Atitle+LIKE+%27My+Title%25%27%27
+```
+
+```json
+{
+  "entity-type": "bulkStatus",
+  "commandId": "90037d73-ed19-48cc-a4ad-09f3999cf014",
+  "state": "SCHEDULED",
+  "processed": 0,
+  "error": false,
+  "errorCount": 0,
+  "total": 0,
+  "action": "index",
+  "username": "system",
+  "submitted": "2019-07-26T09:26:08.761Z",
+  "scrollStart": null,
+  "scrollEnd": null,
+  "processingStart": null,
+  "processingEnd": null,
+  "completed": null,
+  "processingMillis": 0
+}
+```
+
+## Reindex a Document and Its Children Recursively
+
+```
+POST /management/search/DOC_ID/reindex
+```
+
+### Path Parameters
+
+| Parameter Name      | Type       | Description          |
+| ------------------- | ---------- | -------------------- |
+| **DOC_ID**          | **string** | The document Id.     |
+
+### Response
+
+If successful, returns a [bulk status entity]({{page page='bulk-status-entity-type'}}) representing the bulk action status of the index bulk action.
+
+The index status can be monitored using the [Bulk Endpoint]({{page page='bulk-endpoint'}}).
+
+### Status Codes
+
+- 200 *OK* - Success.
+
+### Sample
+
+```curl
+curl -X POST -u Administrator:Administrator \
+http://localhost:8080/nuxeo/api/v1/management/search/1fa9d3fa-04cc-4956-bc6c-8317b803e131/reindex
+```
+
+```json
+{
+  "entity-type": "bulkStatus",
+  "commandId": "f37503d3-51b2-462e-8f1e-fa240fafa861",
+  "state": "SCHEDULED",
+  "processed": 0,
+  "error": false,
+  "errorCount": 0,
+  "total": 0,
+  "action": "index",
+  "username": "system",
+  "submitted": "2019-07-26T09:45:09.874Z",
+  "scrollStart": null,
+  "scrollEnd": null,
+  "processingStart": null,
+  "processingEnd": null,
+  "completed": null,
+  "processingMillis": 0
+}
+```
+
+## Check for search Desynchronisation between Elastic and the Repository
+
+```
+GET /management/search/checkSearch
+```
+
+### Query Parameters
+
+| Parameter Name | Type       | Description                         | Notes                                           |
+|----------------|------------|-------------------------------------|-------------------------------------------------|
+| **nxql**       | **string** | The NXQL query.                     | Optional, no query means all visible documents  |
+| **pageSize**   | **number** | The number of documents to return.  | Optional, default is 10                         |
+
+### Response
+
+If successful, returns a JSON response containing information about the search results from 2 page providers: `repository/repository` and
+`opensearch/enhanced`.
+
+The `repoistory/repository` result is run against repository Search Client on the repository indexbackend using `nxql_repo_search` page provider, while the `opensearch/enhanced` is run against `nxql_elastic_search` page provider.
```

Comment:
In fact, both SearchIndex is queried with `search_check_nxql` PageProvider. Also, the endpoint runs the check against all configured SearchIndex, that ends currently with `repository/repository` and `opensearch/enhanced`.

Could you rephrase the sentence?